### ISSUE TITLE
Rule parse error in rule python.sqlalchemy.security.audit.avoid-sqlalchemy-text.avoid-sqlalchemy-text

### ISSUE THREAD

**Issue by jonny64** [semgrep/semgrep#9617]
**Describe the bug**
{
  "$schema": "https://docs.oasis-open.org/sarif/sarif/v2.1.0/os/schemas/sarif-schema-2.1.0.json",
  "runs": [
    {
      "invocations": [
        {
          "executionSuccessful": true,
          "toolExecutionNotifications": [
            {
              "descriptor": {
                "id": "Rule parse error"
              },
              "level": "error",
              "message": {
                "text": "Rule parse error in rule python.sqlalchemy.security.audit.avoid-sqlalchemy-text.avoid-sqlalchemy-text:\n unexpected key metavariable-type"
              }
            }
          ]
        }
      ],
      "results": [],
      "tool": {
        "driver": {
          "name": "semgrep",
          "rules": [],
          "semanticVersion": "1.24.0"
        }
      }
    }
  ],
  "version": "2.1.0"
}

**To Reproduce**
Steps to reproduce the behavior, ideally a link to https://semgrep.dev:
semgrep scan --error --config p/default --metrics=off --no-git-ignore --sarif -o semgrep.sarif

**Expected behavior**

**Screenshots**

**What is the priority of the bug to you?**

- [ ] P0: blocking your adoption of Semgrep or workflow
- [x ] P1: important to fix or quite annoying
- [ ] P2: regular bug that should get fixed

**Environment**
If not using semgrep.dev: are you running off docker, an official binary, a local build?

**Use case**
What will fixing this bug enable for you?

**Comment by haydenseitz** [comment]
I am getting the same error, specifically when using an [older version of megalinter](https://megalinter.io/6.22.1/descriptors/repository_semgrep/). It looks like this is fixed in at least version 1.33.1 of semgrep (might be earlier). [Megalinter version 7.2.1](https://megalinter.io/7.2.1/descriptors/repository_semgrep/) packages version 1.33.2 of semgrep, and I am not seeing this error when using that version.

EDIT: you can skip the broken rule in older semgrep versions by passing `--exclude-rule python.sqlalchemy.security.audit.avoid-sqlalchemy-text.avoid-sqlalchemy-text`

```shell
semgrep --version
1.16.0

semgrep --config auto --exclude-rule python.sqlalchemy.security.audit.avoid-sqlalchemy-text.avoid-sqlalchemy-text
```

**Comment by jonny64** [comment]
Upgraded
